specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2017-12-01-preview/mysql.json

```diff
@@ -2272,6 +2444,68 @@
         }
       },
       "description": "An error response from the Batch service."
+    },
+    "ServerAdministratorProperties": {
+      "properties": {
+        "administratorType": {
+          "type": "string",
+          "description": "The type of administrator.",
+          "enum": [
+            "ActiveDirectory"
```

Comment:
This enum implies that there might be other types in the future, but the schema is not polymorphic. If you plan to support future types, you may want to use a 'discriminator' and use a unique definition object bound to the enum value.

Reply:
Discussed with the team and there will not be any other type in future.  So this is a won't fix